stl/inc/chrono

```diff
@@ -5442,6 +5441,20 @@ namespace chrono {
 
         return _Os;
     }
+
+    template <class _CharT, class _Traits>
+    basic_ostream<_CharT, _Traits>& operator<<(basic_ostream<_CharT, _Traits>& _Os, const month& _Val) {
+        if (_Val.ok()) {
+            // tm_mon is [0, 11] while month is [1, 12]
+            const tm _Time = {.tm_mon = static_cast<int>(static_cast<unsigned int>(_Val)) - 1};
+            return _Os << _STD put_time(&_Time, _STATICALLY_WIDEN(_CharT, "%b"));
+        }
+
+        if (static_cast<unsigned int>(_Val) < 10) {

```

Comment:
I am slightly curious how that could happen. How can a mont in the range 0-9 be invalid?

Reply:
`0` is invalid.

Reply:
Should we then compare agains 0?

Reply:
Upon rereading the spec, this is an error to have.